packages/core/src/cli.ts

```diff
@@ -91,6 +91,7 @@ prog
     }
 
     if (opts.client) {
+      // TODO: bail out when the user is using gql.tada
```

Comment:
My thinking here is basically that we can check whether

1. We have a valid `gql.tada` installation
2. We have the config set in the `tsconfig`

If the above don't hold up we need to codegen to support users with prior generated code